packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,7 +71,16 @@ export const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) =>
 					<InspectorAdvancedControls>
 						<TextControl
 							label={ __( 'HTML Anchor' ) }
-							help={ __( 'Anchors lets you link directly to a section on a page.' ) }
+							help={ (
+							    <>
+							        { __( 'Enter a word or two — without spaces — to make a unique web address just for this heading, called an "anchor." Then, you’ll be able link directly to this section of your page.' ) }
```

Comment:
```suggestion
							        <p>
							        	{ __( 'Enter a word or two — without spaces — to make a unique web address just for this heading, called an "anchor." Then, you’ll be able link directly to this section of your page.' ) }
							        </p>
```

> I'd like to add 1em of space between paragraphs, but there's currently no CSS file for adding new styles to this section. How should I approach this?

Wrapping this in `<p>` tags should take care of it. This mimics the approach used for the permalink help text: 

https://github.com/WordPress/gutenberg/blob/2ab29100b413feb4932cbd576c9e7a9bde31c18e/packages/edit-post/src/components/sidebar/post-link/index.js#L89-L94

Reply:
Whoops — the commit suggestion is out of date, but the method still stands. 😄